lib/identity_doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -35,7 +39,11 @@ def to_h
             exception: exception,
             result: result_code.name,
             billed: result_code.billed,
+            processed_alerts: processed_alerts,
+            alert_failure_count: processed_alerts[:failed].count,
```

Comment:
ditto about the nil-safeness here:

```suggestion
            alert_failure_count: processed_alerts[:failed]&.count.to_i,
```

Reply:
Doh. I thought about that and then forgot to make the change. Thanks.